saltedge-compliance-connector-java-v2/compliance-connector-sdk/src/main/java/com/saltedge/connector/sdk/api/models/ProviderOfferedConsent.java

```diff
@@ -20,13 +20,15 @@
  */
 package com.saltedge.connector.sdk.api.models;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.saltedge.connector.sdk.SDKConstants;
+import org.springframework.util.StringUtils;
 
 import java.util.Objects;
 
-@JsonIgnoreProperties
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Are you sure it is necessary to add the @JsonIgnoreProperties(ignoreUnknown = true) flag here? By ignoring unknown fields, we won't receive an error, but it might lead to missing data issues. Shouldn't we be more cautious about ensuring all relevant fields are captured and properly handled? This approach could potentially hide issues that need to be addressed in the future.

Reply:
I'm sure. POJO is simple and no potential issues expected. I'm hiding from deserialiser public method.